### PR TITLE
Add steps for Route53 LB to cross-site ROSA

### PR DESCRIPTION
**Pull request by ahus1**
Those hints were missing and cost me some extra time when creating the cluster today.

### CODE REVIEW

**Comment by ahus1** [comment]
@ryanemerson - please review and merge when you have the time. Thanks!

cc: @mhajas

File: doc/kubernetes/modules/ROOT/pages/openshift/cross-site-rosa.adoc [hunk 1]
```diff
@@ -22,8 +22,12 @@ Both Infinispan and Keycloak are deployed in the same namespace.
 
 . Check out the Git repository from https://github.com/keycloak/keycloak-benchmark/.
 
+. Create a Route53 entry as described in xref:./installation-route53-loadbalancer.adoc[].
```
Comment (ryanemerson) [inline doc/kubernetes/modules/ROOT/pages/openshift/cross-site-rosa.adoc:25]: This document details how to use the various *.sh scripts directly, but we now have Tasks for this which greatly simplify things. Should we update this document as well? I'm happy for this to be done as a separate issue.

Comment (ahus1) [reply doc/kubernetes/modules/ROOT/pages/openshift/cross-site-rosa.adoc:25]: I wasn't aware of the taskfile integration. Please create a follow-up issue for this one.

If everything else is fine, please approve and merge. Thanks!

Comment (ryanemerson) [reply doc/kubernetes/modules/ROOT/pages/openshift/cross-site-rosa.adoc:25]: I've created https://github.com/keycloak/keycloak-benchmark/issues/826 to track the work.

**Comment by ryanemerson** [comment]
Thanks @ahus1

**Pull request merged**